.github/scripts/end2end/kustomization.yaml

```diff
@@ -46,13 +46,13 @@ images:
   newTag: 0.34.0-debian-11-r24
 - name: metalk8s-registry-from-config.invalid/zenko-base-2.3.11/mongodb-sharded
   newName: docker.io/bitnami/mongodb-sharded
-  newTag: 5.0.18-debian-11-r15
+  newTag: 6.0.10-debian-11-r8
 - name: metalk8s-registry-from-config.invalid/zenko-base-2.3.18/bitnami-shell
   newName: docker.io/bitnami/bitnami-shell
   newTag: 10-debian-10-r404
```

Comment:
* bitnami shell changed as well
* it would be better to automatically "pick up" these values from solution-base, somehow...